### DISCUSSION TITLE
Bug: cannot analyze project with `\` in path

### DISCUSSION THREAD

**Discussion by yanyongyu** [microsoft/pylance-release#3238] [Enhancement]
## Environment data

-   Language Server version: 2022.8.30
-   OS and version: Ubuntu 22.04
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10

## Repro Steps

1. Open an python project which path containing `\`: e.g. `/media/DOMAIN\USER/projects/foo`
2. Pylance start to analyze: `Searching for source files`
3. Pylance report error: `File or directory "/media/DOMAIN/USER/projects/foo" does not exist`

## Expected behavior

To analyze the project correctly.

## Actual behavior

It seems that the project path reported in the logs is uncorrect. Project analysis failed.

## Logs

```
[Info  - 13:50:23] (95672) Assuming Python version 3.10
[Info  - 13:50:23] (95672) Assuming Python platform Linux
[Info  - 13:50:23] (95672) Searching for source files
[Error - 13:50:23] (95672) File or directory "/media/DOMAIN/USER/Projects/foo" does not exist.
[Info  - 13:50:23] (95672) No source files found.
```

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by yanyongyu** [comment]
Hi, thanks for your review. I don't think this issue is an enhancement request but an issue of project analysis. All projects under my AD account folder `/media/DOMAIN\USER/` could not be analyzed, and import errors reported by pylance are full of my screen.:cry:
